app/host/templates/host/results.html

```diff
@@ -8,9 +8,9 @@
 {% block body %}
 
 <div class="p-5 text-left bg-light">
-  <h1 class="mb-3">{{transient.name}}</h1>
+  <h1 class="mb-3">{{transient.name}} test </h1>
```

Comment:
Nice work, I think you need to remove the word test from this line though  :)

Reply:
I believe I fixed both issues and merged the changes successfully! let me know if you see it on your end.